Subject: [PATCH] DefaultPermissionGrantPolicy: Pass correct arguments

After parsing the xml files in the "default-permissions"
system, we are passing the arguments in the wrong order.

The method we are targeting is:
private void grantRuntimePermissionsLPw(PackageParser.Package pkg, Set<String> permissions,
            boolean systemFixed, boolean isDefaultPhoneOrSms, int userId) {

However we are passing our argument for "systemFixed"
to "isDefaultPhoneOrSms" and the other way around too.

This patch fixes it by using the version of the method
which takes 4 arguments and sets "isDefaultPhoneOrSms"
as false.

diff --git a/services/core/java/com/android/server/pm/DefaultPermissionGrantPolicy.java b/services/core/java/com/android/server/pm/DefaultPermissionGrantPolicy.java
index 5016ec0..a6f9243 100644
--- a/services/core/java/com/android/server/pm/DefaultPermissionGrantPolicy.java
+++ b/services/core/java/com/android/server/pm/DefaultPermissionGrantPolicy.java
@@ -1013,7 +1013,7 @@
                         permissions.clear();
                     }
                     permissions.add(permissionGrant.name);
-                    grantRuntimePermissionsLPw(pkg, permissions, false,
+                    grantRuntimePermissionsLPw(pkg, permissions,
                             permissionGrant.fixed, userId);
                 }
             }
